### PR TITLE
initial test of fr build instructions

### PR DESCRIPTION
**Pull request by jodygarnett**
Looking at sphinx-build internationalization build chain as a follow up to https://github.com/metadata101/iso19139.ca.HNAP/issues/183 discussion.

* [x] stage `rst` contents into ``po`` files for translation
* [x] sphinx-build of ``fr`` documentation
* [x] sort out how to translating an image
* [x] translate a sample page
* [x] review and feedback

Directory outline:

* ``src/sphinx/index.rst`` contains original document
* ``src/sphinx/img/foo.png`` example image
* ``src/sphinx/img/foo_fr.png`` translated image
* ``src/locale/fr/LC_MESSAGES/index.po`` contains ``gettext`` portable object files for translation

See ``README.md`` for manual build steps, at the end of the day this will be part of the maven build.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -124,6 +124,58 @@ jenkins:
...
+### Internationalization
+
+Before you start:
+```
+pip install sphinx-intl
+```
+
+Translation workflow:
+
+1. Geneate `pot` files:
```
Comment (MichelGabriel) [inline README.md]: A small typo here

**Comment by MichelGabriel** [comment]
Found 1 typo, the rest is working as expected.

Would be nice to have a sort of language switch on the pages, or maybe just a link in the TOC

**Comment by jodygarnett** [comment]
> Would be nice to have a sort of language switch on the pages, or maybe just a link in the TOC

Consider that a request for the theme, it will have to be at the theme level each document is build into distinct folders.

**Comment by jodygarnett** [comment]
Okay so this is looking okay, I am focusing on the content translation and making sure it support pictures.

Relative paths between base folder and `fr` folder work:

image

Translating of content with images also works:

image

image

**Comment by jodygarnett** [comment]
One interesting thing from the sample document is the wish to have some readable metadata: overview, process, related, before you start.

I have captured this as directives for now (as headings would interrupt the table of contents outline) but there should be some ability to use sphinx index directives here.

image

**Comment by jodygarnett** [comment]
Okay, merging this in and can experiment with transfix next

**Pull request merged**